src/components/theme-switcher/index.tsx

```diff
@@ -8,23 +8,42 @@ import { IconCaret16 } from '@hashicorp/flight-icons/svg-react/caret-16'
 import { IconMonitor16 } from '@hashicorp/flight-icons/svg-react/monitor-16'
 import { IconMoon16 } from '@hashicorp/flight-icons/svg-react/moon-16'
 import { IconSun16 } from '@hashicorp/flight-icons/svg-react/sun-16'
-import { useState, useEffect } from 'react'
+import classNames from 'classnames'
+import { useState, useEffect, ReactElement } from 'react'
```

Comment:
🎨 Should we adjust import order while we're here?

Reply:
What order were you thinking? I always think 'external' at the top.

Reply:
Yeah same! And then within "external" we've commonly done React, then Next, then totally external (like classnames), then HashiCorp (like flight)

Reply:
ah okay, I'll reorder.